susemanager-utils/susemanager-sls/salt/cleanup_ssh_minion/init.sls

```diff
@@ -47,3 +47,8 @@ mgr_remove_own_ssh_pub_key:
 mgr_remove_own_ssh_key:
   file.absent:
     - name: {{ home }}/.ssh/mgr_own_id
+
+# Remove logrotate configuration
+mgr_remove_logrotate_configuration:
+  file.absent:
+    - name: /etc/logrotate.d/salt-ssh
```

Comment:
We remove the salt-ssh config but NOT the log.  That's considered a "user file", and most packages will not remove it.